tests/simpleKVBC/TesterReplica/internalCommandsHandler.cpp

```diff
@@ -41,6 +41,17 @@ int InternalCommandsHandler::execute(uint16_t clientId,
                                      uint32_t &outActualReplySize,
                                      uint32_t &outActualReplicaSpecificInfoSize,
                                      concordUtils::SpanWrapper &span) {
+  bool wedgeCommand = flags & 0x9;
```

Comment:
`0x9` doesn't really work as a flag, as it uses two bits: bit 0, and bit 3. If someone added a flag of `0x8`, then `0x8` and `0x1` (read-only) combined would match a wedge command and lead to incorrect behavior. You should just use `0x4` or `0x8` instead.

Reply:
I have removed the flag completely (as wedge command should be a write command)